samples/Ralltiir/Ralltiir.Server/Controllers/AuthorizationController.cs

```diff
@@ -0,0 +1,397 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/openiddict/openiddict-core for more information concerning
+ * the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.Extensions.Primitives;
+using OpenIddict.Abstractions;
+using OpenIddict.Core;
+using OpenIddict.EntityFrameworkCore.Models;
+using OpenIddict.Server.AspNetCore;
+using OpenIddict.Validation.AspNetCore;
+using Ralltiir.Server.Extensions;
+using Ralltiir.Server.Helpers;
+using Ralltiir.Server.Models;
+using Ralltiir.Server.ViewModels;
+using static OpenIddict.Abstractions.OpenIddictConstants;
+
+namespace Ralltiir.Server.Controllers
+{
+    public class AuthorizationController : Controller
+    {
+        private readonly SignInManager<ApplicationUser> _signInManager;
+        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly OpenIddictApplicationManager<OpenIddictEntityFrameworkCoreApplication<Guid>> _applicationManager;
+        private readonly OpenIddictAuthorizationManager<OpenIddictEntityFrameworkCoreAuthorization<Guid>> _authorizationManager;
+        private readonly OpenIddictScopeManager<OpenIddictEntityFrameworkCoreScope<Guid>> _scopeManager;
+
+        public AuthorizationController(
+            SignInManager<ApplicationUser> signInManager,
+            UserManager<ApplicationUser> userManager,
+            OpenIddictApplicationManager<OpenIddictEntityFrameworkCoreApplication<Guid>> applicationManager,
+            OpenIddictAuthorizationManager<OpenIddictEntityFrameworkCoreAuthorization<Guid>> authorizationManager,
+            OpenIddictScopeManager<OpenIddictEntityFrameworkCoreScope<Guid>> scopeManager)
+        {
+            _signInManager = signInManager;
+            _userManager = userManager;
+            _applicationManager = applicationManager;
+            _authorizationManager = authorizationManager;
+            _scopeManager = scopeManager;
+        }
+        
+        [IgnoreAntiforgeryToken]
+        [HttpGet("~/oidc")]
+        [HttpPost("~/oidc")]
+        public IActionResult OidcCallback() => Ok();
```

Comment:
Do we need to server-side route for that? Can't you handle that client-side?

Reply:
Oops... didn't mean to commit that 🤦🏻